### PR TITLE
add misc compiler errors and warnings

### PR DESCRIPTION
**Pull request by BillWagner**
Update the message TOC, update the breadcrumbs file

Found while looking at the low PV report. Most of them are in the language reference / compiler messages area.

[Internal review site](https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/?branch=pr-en-us-23968)

### CODE REVIEW

File: docs/csharp/language-reference/compiler-messages/toc.yml [hunk 1]
```diff
@@ -3,218 +3,1550 @@
   items:
   - name: Compiler error CS0001
     href: cs0001.md
+  - name: Compiler Error CS0003
```
Comment (gewarren) [inline docs/csharp/language-reference/compiler-messages/toc.yml:6]:
```suggestion
  - name: Compiler error CS0003
```

File: docs/csharp/language-reference/compiler-messages/toc.yml [hunk 2]
```diff
@@ -3,218 +3,1550 @@
   items:
```
Comment (gewarren) [inline docs/csharp/language-reference/compiler-messages/toc.yml:3]: Maybe have two top level nodes: Errors and Warnings. Then nest the individual messages underneath, omitting either "Compiler error" or "Compiler warning" from the displayed text. As is, it's one very long list, and a lot of repeated words.

Comment (BillWagner) [reply docs/csharp/language-reference/compiler-messages/toc.yml:4]: Great suggestion!

File: docs/csharp/language-reference/compiler-messages/toc.yml (hunk 1 above)
Comment (BillWagner) [reply docs/csharp/language-reference/compiler-messages/toc.yml:7]: replaced in all locations (and warnings)

**Comment by BillWagner** [comment]
Thanks @gewarren for the suggestion.  It's a big improvement.

This should be ready for final review.

**Review by gewarren — APPROVED** [review]
Looks much better! If you want to take it a step further, https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/?branch=pr-en-us-23968 should also be updated:

- The title should be "C# compiler messages"
- It's very VS focused and assumes you're using VS
- It should recommend putting the compiler message number in the TOC filter to find the topic

**Pull request merged**